.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: ''
```

Comment:
can we add default labels? It would be helpful to tag any bug with a "bug" label upfront.

Reply:
sure